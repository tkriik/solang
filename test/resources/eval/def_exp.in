(def foo (head '(a b c)))
foo
foo

(def bar (tail '(a b c)))
bar
bar
