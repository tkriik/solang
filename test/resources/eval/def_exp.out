foo
a
a

bar
(b c)
(b c)
